### DISCUSSION TITLE
We should suggest this as working with CustomTkinter

### DISCUSSION THREAD

**Discussion by RedLeafSystems** [rdbende/Sun-Valley-ttk-theme#76] [Polls]
I found that even when I used the Sun Valley theme on Tkinter, the output still lacked resolution and didn't look great.
So, my solution to this was to use a [CustomTkinter](https://github.com/TomSchimansky/CustomTkinter) canvas and place the Tkinter widgets that were styled with Sun Valley onto it.  I now have some stunning looking GUIs.
I propose that on the Sun Valley GitHub readme file, we should mention it's compatible with CustomTkinter if you want higher DPI.

**Comment by rdbende** [comment]
Could you show a screenshot? I'm curious, how could CustomTkinter make this work with hi-res.

**Reply by sumeshir26** [reply]
> Could you show a screenshot? I'm curious, how could CustomTkinter make this work with hi-res.

https://github.com/TomSchimansky/CustomTkinter/blob/09e584634c867f2b6074fcbfe41cba80d2b78c66/customtkinter/windows/widgets/core_rendering/draw_engine.py#L11

**Reply by rdbende** [reply]
This doesn't do anything with the ttk widgets.

**Comment by RedLeafSystems** [comment]
Sure! (Just ignore the menubar on top.)
![image](https://user-images.githubusercontent.com/119130769/231522254-5be6ce75-1eba-4c13-9a97-3416a074dcbe.png)
You can't really see how great it looks, but that's because the components are so small.
Here's the code required:
```
import customtkinter
import tkinter
from tkinter import ttk
import sv_ttk

app = customtkinter.CTk()  # create CTk window like you do with the Tk window
button = ttk.Button(app, text="This is a Sun Valley button.")
edit = ttk.Entry(app)

button.grid(column=0, row=0)
edit.grid(column=1, row=0)

app.mainloop()
```

**Comment by HuyHung1408** [comment]
It's just as same as `ctypes.windll.shcore.SetProcessDpiAwareness(True)`, which makes buttons and other widgets super small on high scailing. 
<img width="831" alt="image" src="https://user-images.githubusercontent.com/86362423/231808245-789f2eb9-6af6-40e9-920e-7e8988736ade.png">

Without `ctypes.windll.shcore.SetProcessDpiAwareness(True)`, with the bad font rendering on 150% scaling: 
![image](https://user-images.githubusercontent.com/86362423/231808646-7555d577-b058-434e-9d2c-01852f4f7ad5.png)
